tools/pythonpkg/tests/fast/test_all_types.py

```diff
@@ -42,8 +30,71 @@ def recursive_equality(o1, o2):
         return False
 
 
+# Regenerate the 'all_types' list using:
```

Comment:
Any reason why you're making this static now?

Reply:
I feel like adding to `test_all_types` has way too much direct effect, increasing the threshold of making these changes.

This allows us to roll out these changes more gradually

Reply:
Happy to revert it if you strongly disagree, but it's trivial to regenerate and now a PR can be made to *only* add 'test_ll_types' support for a specific type in Python, instead of clumping every client and call to `test_all_types` together in one PR.